src/core/components/Dropzone/Dropzone.tsx

```diff
@@ -36,29 +41,29 @@ const Dropzone = (props: DropzoneProps) => {
     onChange,
   } = props;
   const { t } = useTranslation();
-  const [isMounted, setMounted] = useState(false);
-  const [inputKey, setInputKey] = useState("");
-  const { getInputProps, getRootProps, acceptedFiles, fileRejections } =
+  const [inputKey] = useState("");
+  const [acceptedFilesMap, setAcceptedFilesMap] = useState<
+    Map<string, FileWithPath>
+  >(new Map());
+  const acceptedFiles = acceptedFilesMap.values().toArray();
+  const { getInputProps, getRootProps, fileRejections, isDragAccept } =
```

Comment:
The state from Dropzone is not considering previously added files -> maintain our own state